jvm/src/main/kotlin/app/cash/trifle/delegate/EndEntityDelegate.kt

```diff
@@ -0,0 +1,14 @@
+package app.cash.trifle.delegate
+
+import app.cash.trifle.protos.api.alpha.MobileCertificateRequest
+
+interface EndEntityDelegate {
```

Comment:
right now the only function this implements is used for testing, so it's questionable as to whether or not we need this.  I think keeping it is ok, since we could easily get a service (not android client) that wants to create a cert request.

Reply:
I thought the point was for android to be able to use the jvm code as well